native/packages/react-native-bpk-component-boilerplate/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "react-native-bpk-component-boilerplate",
+  "version": "0.0.0",
```

Comment:
should this have `private: true` set?

Reply:
I think we're ok without it. The web one doesn't have it, and it's ignored in `lerna.json` so it can't actually get published.

Reply:
Tru dat!